plugin/procfs/tests/mtr/procfs_basic.result

```
@@ -0,0 +1,14 @@
+INSTALL PLUGIN procfs SONAME 'procfs.so';
+CREATE USER mysqltest_1@localhost;
+GRANT ALL PRIVILEGES ON mysqltest.* TO mysqltest_1@localhost;
+GRANT ACCESS_PROCFS ON *.* TO 'mysqltest_1'@'localhost';
+SELECT COUNT(*) FROM information_schema.procfs WHERE file = '/proc/version';
+COUNT(*)
+0
```

why is this 0, shouldn't /proc/version exists?

---

another good catch, I suspect the default access mask is not allowing access to version, need to investigate.